### DISCUSSION TITLE
There are currently problems with the linux py311 CI builds

### DISCUSSION THREAD

**Discussion by greglandrum** [rdkit/rdkit#6478] [Development]
@bp-kelley @ricrogz @ptosco 
Starting a few days ago the linux python 3.11 CI builds started failing in two completely unrelated (as far as I can tell) tests.

The only configuration differences that I can see between a working build:
https://dev.azure.com/rdkit-builds/RDKit/_build/results?buildId=3755&view=logs&jobId=3f8dec1d-898f-527d-826e-9d8291e77ed4&j=3f8dec1d-898f-527d-826e-9d8291e77ed4&t=f3ba3d3e-9cbc-56bc-12cf-d886dd652389

and a failing one:
https://dev.azure.com/rdkit-builds/RDKit/_build/results?buildId=3762&view=logs&jobId=3f8dec1d-898f-527d-826e-9d8291e77ed4&j=3f8dec1d-898f-527d-826e-9d8291e77ed4&t=38d2d8b4-6132-5062-8b90-dd3f16d66919

is an update to the numpy and pyparsing versions. 

I'm not even sure why pyparsing is being installed (everything worked fine for me without installing it), but one could imagine the numpy version making a difference. However, when I set up a similar environment on my linux build box everything works fine.

I will keep trying to track this down, but I'm going to be out of office (with laptop, but not my linux box) for a couple of weeks, so it probably won't go fast. Unless one of you has time to investigate and figure it out, I think we're going to have to live with having these tests fail for a bit. With each PR you can just look at the specific failures and ignore them if they are in `pythonTestDirML` or `github4823`

**Comment by bp-kelley** [comment]
I'll take a look.
